core/commands/publish.go

```diff
@@ -89,7 +91,8 @@ Publish an <ipfs-path> to another public key (not implemented):
 		}
 
 		// TODO n.Keychain.Get(name).PrivKey
-		output, err := publish(n, n.PrivateKey, p)
+		// TODO(cryptix): is req.Context().Context a child of n.Context()?
```

Comment:
@jbenet ohai cmdlib :disappointed: .. but fixed :smile:. Do I need to mirror this change on the offline mode, too?